earth_enterprise/src/fusion_version.txt

```diff
@@ -20,5 +20,5 @@
 # Format is 2 lines (comment lines ignored):
 # 1st) full version number: e.g., 3.1.2.3,
 # 2nd) is short number (i.e., drop trailing 0's, i.e., 3.1.0 -> 3.1, or 4.0.0 -> 4)
-5.1.3
-5.1.3
+5.2.0
+5.2.0
```

Comment:
based on comments , should this be short form 5.2 ? either that or comments need a change.